feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -248,7 +248,6 @@ decapsulate MPLS in UDP.  # TODO: Move to dedicated README
 openconfig-network-instance:
   network-instances:
     - network-instance: "DEFAULT"
-      afts:
```

Comment:
This is actually ok.  the path is:


/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/config/decapsulate-mpls-in-udp
--


<br class="Apple-interchange-newline">[/network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/config/decapsulate-mpls-in-udp](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-policy-forwarding-policies-policy-rules-rule-action-config-decapsulate-mpls-in-udp)